src/dashboard_api/__init__.py

```diff
@@ -1,189 +0,0 @@
-import os
-import uuid
-
-import eventlet
-eventlet.monkey_patch()
-
-import flask_socketio
-import socketio
-
-from flask import jsonify
-
-from pyinfraboxutils import get_env, print_stackdriver, get_logger
-from pyinfraboxutils.ibrestplus import app, api
-from pyinfraboxutils.ibflask import get_token, is_collaborator
-from pyinfraboxutils import dbpool
-
-# TODO: do it the same way in api
-import dashboard_api.handlers
-
-import dashboard_api.listeners.job
-import dashboard_api.listeners.console
-
-logger = get_logger('dashboard-api')
-
-# TODO: Move to common
-class ClientManager(socketio.base_manager.BaseManager):
-    def __init__(self):
-        super(ClientManager, self).__init__()
-        self.__rooms = {}
-
-    def enter_room(self, sid, namespace, room):
-        super(ClientManager, self).enter_room(sid, namespace, room)
-        logger.debug('%s joined room %s', sid, room)
-
-        if room not in self.__rooms:
-            self.__rooms[room] = 0
-
-        self.__rooms[room] += 1
-
-    def leave_room(self, sid, namespace, room):
-        super(ClientManager, self).leave_room(sid, namespace, room)
-        logger.debug('%s left room %s', sid, room)
-        self.__rooms[room] -= 1
-
-        if not self.__rooms[room]:
-            del self.__rooms[room]
-
-    def has_clients(self, room):
-        clients = self.__rooms.get(room, None)
-
-        if clients:
-            return True
-
-        return False
-
-
-
-def main(): # pragma: no cover
-    get_env('INFRABOX_SERVICE')
-    get_env('INFRABOX_VERSION')
-    get_env('INFRABOX_DATABASE_HOST')
-    get_env('INFRABOX_DATABASE_USER')
-    get_env('INFRABOX_DATABASE_PASSWORD')
-    get_env('INFRABOX_DATABASE_PORT')
-    get_env('INFRABOX_DATABASE_DB')
-
-    get_env('INFRABOX_GITHUB_ENABLED')
-    get_env('INFRABOX_GERRIT_ENABLED')
-    get_env('INFRABOX_ACCOUNT_SIGNUP_ENABLED')
-    get_env('INFRABOX_ACCOUNT_LDAP_ENABLED')
-    get_env('INFRABOX_ROOT_URL')
-    get_env('INFRABOX_GITHUB_LOGIN_ENABLED')
-
-    app.config['MAX_CONTENT_LENGTH'] = 1024 * 1024 * 1024
-
-    client_manager = ClientManager()
-    sio = flask_socketio.SocketIO(app,
-                                  path='/api/dashboard/socket.io',
-                                  async_mode='eventlet',
-                                  client_manager=client_manager)
-
-    @sio.on('connect')
-    def __connect():
-        try:
-            get_token()
-        except:
-            logger.debug('disconnecting connection')
-            return False
-
-        return True
-
-    @sio.on('listen:jobs')
-    def __listen_build(project_id):
```

Comment:
did you move the socket io handlers?

Reply:
Yes

Reply:
I don't see it. And be carefule the implementations are different on api and dashboard_api